app/controllers/dishes_controller.rb

```diff
@@ -2,8 +2,13 @@ class DishesController < ApplicationController
   load_and_authorize_resource
 
   def new
-    @dish = Dish.new
-    @menus = Menu.all # Later on, we need to restrict this to only the menus of the current Owner
+    if (can? :manage, Dish) && Restaurant.exists?(user: current_user)
```

Comment:
I would like us to visit this line together. Not sure about `Restaurant.exists?(user: current_user)`